src/hooks/useFocusRef.tsx

```diff
@@ -5,7 +5,7 @@ export function useFocusRef<T extends HTMLOrSVGElement>(isCellSelected: boolean
   useLayoutEffect(() => {
     if (!isCellSelected) return;
     ref.current?.focus({ preventScroll: true });
-  }, [isCellSelected]);
+  });
```

Comment:
Won't that always try and focus the cell? Even when we don't interact with the grid?

Reply:
Only if the formatter is rerendered which should not happen if the grid props have not changed. But in the [TreeView ](https://adazzle.github.io/react-data-grid/canary/?path=/story/demos--tree-view) example it steals the focus when `Allow Delete` is clicked. Need to think more. Great catch